syntax = "proto3";

package myservice;

service MyService {
  rpc GetUser (UserRequest) returns (UserResponse);
}

message UserRequest {
  int32 user_id = 1;
}

message UserResponse {
  int32 user_id = 1;
  string username = 2;
  string email = 3;
}